portafly/src/i18n/locales/en/product.yml

```diff
@@ -0,0 +1,25 @@
+---
+# ------------------ Product > Edit product
+# Page details
+
+page_title: Edit product #--- PROPOSED
+page_title_description: Page title to edit product details
+body_title: "Edit product: {{product_name}}" #--- PROPOSED
+body_title_description: Body title of the page to edit product details
+
+# Main area
+
+name: Name
+name_description: Field label to show/input the name of the product.
+system_name: System name
```

Comment:
@porueesq @dfennessy As per #2097 we have added inline help to the System name field to explain to users that a System name cannot be changed.

Sorry for last minute change!

Reply:
Working on this @cntlsn .